### DISCUSSION TITLE
Urgent Inquiry: Using WebView2 with Microsoft Edge AppData

### DISCUSSION THREAD

**Discussion by divyadesh** [MicrosoftEdge/WebView2Feedback#4419] [Q&A] [answered]
Hi Team,

I am using WebView2 for Win32 applications, and I am looking to implement a functionality where WebView2 utilizes the AppData of Microsoft Edge.

Is there any way to accomplish this?

I have attempted to achieve this with the following code:

```
LPCWSTR dataFolder = L"C:\\Users\\Username\\AppData\\Local\\Microsoft\\Edge";

CreateCoreWebView2EnvironmentWithOptions(nullptr, dataFolder, nullptr,
    Callback<ICoreWebView2CreateCoreWebView2EnvironmentCompletedHandler>(
        [hWnd](HRESULT result, ICoreWebView2Environment* env) -> HRESULT {
            // Your callback function code here
        });
```

However, I have observed that CreateCoreWebView2EnvironmentWithOptions appends "EBWebView" at the end of the provided dataFolder location. Therefore, I am seeking to know if there is any alternative function or method available that can achieve this functionality.

Your prompt response would be greatly appreciated, as I require this change urgently.

Thank you.

**Comment by victorhuangwq** [comment, marked as answer]
This is not possible by design. Please refer to previous similar discussions and issues below:
https://github.com/MicrosoftEdge/WebView2Feedback/discussions/4385
https://github.com/MicrosoftEdge/WebView2Feedback/issues/2034

**Reply by divyadesh** [reply]
Thank you @victorhuangwq
